great_ape_safe/ape_api/rari.py

```diff
@@ -59,3 +59,16 @@ def add_ftoken_to_pool(self, ftoken_addr, cf=None):
         else:
             self.unitroller._supportMarket(ftoken_addr)
         assert ftoken_addr in self.unitroller.getAllMarkets()
+
+
+    def upgrade_comptroller(self, implementation):
+        self.unitroller._setPendingImplementation(implementation)
+        assert self.unitroller.pendingComptrollerImplementation() == implementation
+        self.unitroller._acceptImplementation()
```

Comment:
I believe this needs to be called instead from the new implementation, [ref](https://etherscan.io/address/0xe3952d770FB26CC61877CD34Fbc3A3750881e9A1#code#F23#L63)

method: https://etherscan.io/address/0xe16db319d9da7ce40b666dd2e365a4b8b3c18217#code#F9#L1406

Reply:
yup, I believe this is right, that would make sense as to why there are separate methods to set pending implementation and accepting it